data/json/recipes/tools/tools_hand.json

```diff
@@ -33,6 +33,26 @@
     "tools": [ [ [ "hotcut", -1 ] ] ],
     "components": [ [ [ "plastic_chunk", 2 ], [ "2x4", 1 ], [ "stick", 2 ] ] ]
   },
+  {
+    "result": "flint_steel",
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "time": "2 h",
+    "autolearn": true,
+    "using": [ [ "forging_standard", 2 ] ],
+    "proficiencies": [
+      { "proficiency": "prof_metalworking" },
+      { "proficiency": "prof_blacksmithing" },
+      { "proficiency": "prof_toolsmithing" }
+    ],
+    "qualities": [ { "id": "ANVIL", "level": 3 }, { "id": "HAMMER", "level": 3 } ],
+    "tools": [ [ [ "tongs", -1 ] ] ],
+    "components": [ [ [ "hc_steel_chunk", 1 ] ], [ [ "sharp_rock", 1 ] ] ]
```

Comment:
Ehhhhhh..... sharp rock isn't specifically flint, it's just any conchoidal stone, promoting it to flint is quite a stretch.

Reply:
My search turned up that in #41711 it was renamed into "flint" with new description specifically mentioning that it makes sparks when struck against steel, and even before that, it was used for making flintlock firearms, so I guess it is meant to be flint specifically. Item ID wasn't changed though.

Before this PR it apparently was a generic rock with an edge usable only for extremely low-quality butchering.

The PR also added an item named "flaking rock" which I think is some rock that won't make sparks but could be used for making stone tools via knapping.

I could rename it back to "sharp rock" and add a new rock named "flint", but then I'm not sure what exactly this "sharp rock" would represent.